packages/api-explorer/src/form-components/FileWidget.jsx

```diff
@@ -0,0 +1,14 @@
+const extensions = require('@readme/oas-extensions');
```

Comment:
Isn't there already a way to upload a file in the new api explorer?

Reply:
I added this wrapper so that when `x-explorer-enabled: false` then it doesn't show the <input>